vscode-dotnet-runtime-library/src/Utils/ErrorHandler.ts

```diff
@@ -126,17 +125,15 @@ export async function callWithErrorHandling<T>(callback: () => T, context: IIssu
 
 async function configureManualInstall(context: IIssueContext, requestingExtensionId: string): Promise<void> {
     const manualPath = await context.displayWorker.displayPathConfigPopUp();
-    if (manualPath && fs.existsSync(manualPath))
-    {
-        try
-        {
-            let configVal: IExistingPath[] = [{ [ExistingPathKeys.extensionIdKey]: requestingExtensionId, [ExistingPathKeys.pathKey] : manualPath}];
+
+    if (manualPath && fs.existsSync(manualPath)) {
+        try {
+            let configVal: ILocalExistingPath[] = [{ [ExistingPathKeys.extensionIdKey]: requestingExtensionId, [ExistingPathKeys.pathKey] : manualPath}];

```

Comment:
This was not in the original spec, so I apologize, but I think it would be better here to set the global path here. We failed to install on the users machine which would likely be extension independent, at least more likely than not. So we'd rather them configure it for all extensions in 1 go instead of for just 1 and have to redo this process.
```suggestion
            let configVal: ILocalExistingPath[] = [{ [ExistingPathKeys.extensionIdKey]: requestingExtensionId, [ExistingPathKeys.pathKey] : manualPath}];
```